src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/RemovedInPlatformAttribute.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Runtime.InteropServices
+{
+
+    /// <summary>
+    /// Marks APIs that were removed in a given operating system version.
+    ///
+    /// Primarily used by OS bindings to indicate APIs that are only available in
+    /// earlier versions.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Assembly |
+                    AttributeTargets.Class |
+                    AttributeTargets.Constructor |
+                    AttributeTargets.Event |
+                    AttributeTargets.Method |
+                    AttributeTargets.Module |
+                    AttributeTargets.Property |
+                    AttributeTargets.Struct,
+                    AllowMultiple = true, Inherited = false)]
+    public sealed class RemovedInPlatformAttribute : PlatformAttribute
```

Comment:
@terrajobst Should this be `RemovedInOSPlatformAttribute` or is `RemovedInPlatformAttribute` right?

Reply:
It should be `RemovedInOSPlatformAttribute` https://github.com/dotnet/runtime/issues/33331#issuecomment-650326500